app/assets/stylesheets/spree/frontend/fontello.css

```diff
@@ -61,4 +61,4 @@
 .icon-spree-google-oauth2:before { content: '\e806'; } /* '' */
 .icon-spree-google-oauth2-circled:before { content: '\e807'; } /* '' */
 .icon-spree-amazon:before { content: '\e808'; } /* '' */
-.icon-spree-amazon-circled:before { content: '\e808'; } /* '' */
\ No newline at end of file
+.icon-spree-amazon-circled:before { content: '\e808'; } /* '' */
```

Comment:
We can remove all the references to amazon/facebook/twitter from this file, right?

Reply:
Removed here [4fb5b3d](https://github.com/solidusio-contrib/solidus_social/commit/4fb5b3dd6b0041caae55915836b3c5666fd83ed9)